daemon/api_quotas_test.go

```diff
@@ -203,6 +206,11 @@ func (s *systemsSuite) TestPostQuotaRequiresRoot(c *check.C) {
 }
 
 func (s *apiQuotaSuite) TestListQuotas(c *check.C) {
+	s.SysctlBufs = [][]byte{
```

Comment:
Silly question - given that we mock above `servicestate.MockSystemdVersion(248)` above, this should not be needed, yes?

Reply:
I suspect so

Reply:
yeah this can go away now

Reply:
removed